### ISSUE TITLE
USWDS package 3.0 update

### ISSUE THREAD

**Issue by Dahianna** [GSA/digitalcorps.gsa.gov#463]
Update to this new version. It might fix some bugs

**Comment by Dahianna** [comment]
It seems we have two different compilers (gulp compiler and webpack compiler) and need to select one before we can move forward with the updates. Asking USWDS to help figure out which one we are using currently.

**Comment by Dahianna** [comment]
Package updated, now working on getting css files ready. Hoping to have it done end of week

**Comment by Dahianna** [comment]
Some styles work but other still need work

**Comment by Dahianna** [comment]
Jared working with USWDS person who might know how to solve the issue. Jared shared screenshots with them.

**Comment by Dahianna** [comment]
Jared followed up with USWDS to learn if they've looked at his update. No updates from USWDS team.

We could put on hold and revert to older version for now and get back to it when we have updates from USWDS team. Dahianna will follow up with Dan Williams

**Comment by Dahianna** [comment]
Jared got some help and working on that

**Comment by Dahianna** [comment]
USWDS folks have not seen something like this before and not sure what to do. Their recommendation did not work, Jared looking into other options

**Comment by Dahianna** [comment]
USDS just updated to 3.0 and also uses a Jekyll site. Jared will check their code and compare with ours. If nothing comes up he will ask in the USWDS slack channel.

**Comment by Dahianna** [comment]
Also checking 18F

**Comment by Dahianna** [comment]
Jared asked in USWDS Slack channel and now working with James Mejia and Charlie Mahoney (Both USWDS)
